.github/workflows/spark-periodic-tests.yml

```diff
@@ -0,0 +1,58 @@
+name: Spark Periodic Integration Tests
+
+on:
+  schedule:
+    - cron: "0 4 * * 5" # Run at 4AM PST on every Friday
+  workflow_dispatch:
+
+jobs:
+  run-databricks-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 480
+    name: SDK Integration Tests against Databricks Compute
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ./.github/actions/setup-server
+        timeout-minutes: 5
+
+      # TODO(ENG-2537): Use our separate GH actions credentials.
+      - uses: ./.github/actions/fetch-test-config
+        with:
+          aws_access_key_id: ${{ secrets.KENNY_AWS_ACCESS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.KENNY_AWS_SECRET_ACCESS_KEY }}
+          s3_test_config_path: periodic-databricks-test-config.yml
+
+      - name: Install any data connector packages
+        run: |
+          aqueduct install s3
+          aqueduct install snowflake
+
+      - name: Run the SDK Integration Tests
+        working-directory: integration_tests/sdk
+        run: pytest aqueduct_tests/ -rP -vv -n 1
```

Comment:
Also, do things fail if we crank the concurrency up? It might make things a lot faster (you may have to provision a slightly beefier machine though).

Reply:
Will experiment with this before merging